application/src/main/java/org/togetherjava/tjbot/commands/moderation/modmail/ModMailCommand.java

```diff
@@ -104,10 +107,34 @@ public void onSlashCommand(SlashCommandInteractionEvent event) {
         if (handleIsOnCooldown(userId, event)) {
             return;
         }
+
+        TextInput body = TextInput.create(MESSAGE, "Message", TextInputStyle.PARAGRAPH)
+            .setPlaceholder("Your concerns go here")
+            .setMinLength(5)
+            .setMaxLength(1000)
+            .build();
+
+        String modMailModalComponentId =
+                generateComponentId(event.getOption(OPTION_STAY_ANONYMOUS).getAsString(),
+                        String.valueOf(event.getOption(OPTION_GUILD).getAsLong()));
```

Comment:
bit long to read all of this, please unbox this stuff earlier and seperated. so like:
```java
boolean stayAnonymous = event.getOption(...).getAsBoolean();
long guildToContact = event.getOption(...).getAsLong();

...

String componentId = generateComponentId(
  Boolean.toString(stayAnonymous),
  Long.toString(guildToContact));
```

Reply:
✔️